src/DIRAC/TransformationSystem/Service/TransformationManagerHandler.py

```diff
@@ -33,6 +34,29 @@ def initializeHandler(cls, serviceInfoDict):
 
         return S_OK()
 
+    def checkPermissions(self, transName):
+        """
+        checks if remote user has permission to access to a given transformation
+        :param str transName: Name of the transformation to check
```

Comment:
```suggestion

        :param str transName: Name of the transformation to check
```

(new line before starting with `:param` etc.)